tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_beacon.c

```diff
@@ -229,6 +235,128 @@ static void test_rx_on_key_refresh(void)
 	PASS();
 }
 
+static void test_tx_beacon_init(void)
+{
+	bt_mesh_test_cfg_set(&tx_cfg, BEACON_OBSERVE_INTERVAL);
+}
+
+static void test_rx_beacon_init(void)
+{
+	bt_mesh_test_cfg_set(&tx_cfg, BEACON_OBSERVE_INTERVAL);
+}
+
+static void test_tx_beacon_secure(void)
+{
+	uint32_t delay_time;
+	uint32_t sent_beacons;
+	uint32_t sent_beacon_time;
+	struct bt_mesh_subnet *sub;
+	
+	/* shift beaconing time line to avoid boundary cases. */
+	delay_time = get_device_nbr() * 100;
```

Comment:
Hm. Why do you need it? My assumption is that beacon observation logic should manage the initial beacon storm, is that?

Reply:
Observation logic is OK but, nodes starts perfectly synchronous  and stays that way indefinitely.